Created the Toolbar and headline section

AyoMoses1 opened the pull request:
I was able to create the toolbar and also the headline section of the homepage. I also ensured that I followed best practices in my codebase and also integrated the Figma design of the wireframe provided

Commit:
add linters file to the folder: .github\workflows

Commit:
add .hintrc file to the root folder

Commit:
add .stylelintrc.json file at the root folder

Commit:
add index.html and style.css file at the root folder

Commit:
add toolbar

Commit:
add hero section

Commit:
fix stylelint errors

AyoMoses1 committed:
Merge pull request #2 from AyoMoses1/created_toolbar

Created Toolbar Section

Commit:
fix stylelint errors

AyoMoses1 committed:
Merge pull request #3 from AyoMoses1/created_hero

Created Hero

Commit:
updated readme file

ha-manel reviewed (commented):
# STATUS: INVALID 🔴 
Hi 
I'm sure you put a lot of effort into this project but this PR is invalid because your README file does not follow the new template😓kindly use this [template](https://github.com/microverseinc/readme-template) by copying all of its content and filling out the sections with information related to your project 👍 

Kindly fix the issue and submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
updated readme file

Commit:
updated the readme file

Commit:
updated the readme file with Microverse readme template

Gambit142 commented on .gitignore:
```diff
@@ -0,0 +1,3 @@
+#.gitignore
+node_modules
+test.md
```
- [ OPTIONAL] It would be nice to remove this from the `.gitignore` file. You won't be needing this in this project, so there is no need to keep it in the file. 👍

Gambit142 commented on README.md:
```diff
@@ -1 +1,216 @@
+> Describe 1 - 3 features you will add to the project.
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```
- [ ] Kindly add 1-3 features that can be added to your project. This would help people who want to contribute to your project know which areas they can focus on. 👍

Gambit142 commented on README.md:
```diff
@@ -1 +1,216 @@
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/AyoMoses1)
+- Twitter: [@twitterhandle](https://twitter.com/Ayo_Moses1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ayo-moses-493946184/)
```
- [ OPTIONAL] Great work mapping your social media links to placeholder words. I would like to advise you to use your name or social media usernames instead of @githubhandle or @twitterhandle. For example,

- GitHub: [@AyoMoses1](https://github.com/AyoMoses1) instead of [@githubhandle]. 

The reason why I advise this is because ordinarily if someone sees @githubhandle or @twitterhandle, they would think it is the placeholder name and no link is attached to it. 👍

Gambit142 commented on README.md:
```diff
@@ -1 +1,216 @@
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
- [ ] The link to the _**LICENSE**_ is broken. That is because there is no LICENSE file in your root directory. Kindly add a LICENSE file. 👍

Gambit142 commented on index.html:
```diff
@@ -0,0 +1,37 @@
+<body>
+   <header>
+      <div class="logo">
+         <h3>My Logo</h3>
```
- [ ] Kindly use `a` tag instead of `h3` tag. It is the convention to use `a` tag for logos or title in the header/navigation section of a web page. Using `a` tag is also good for semantic purposes and it provides a great user experience. Kindly read this [article](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more details. 👍

Gambit142 commented on index.html:
```diff
@@ -0,0 +1,37 @@
+      <div class="logo">
+         <h3>My Logo</h3>
+      </div>
+      <div class="hamburger">
```
- [ ] Kindly use a `nav` tag instead of a `div` tag. Using a `nav` tag is more semantic than using a `div` tag to wrap the _**hamburger**_ button. 👍

Gambit142 commented on index.html:
```diff
@@ -0,0 +1,37 @@
+         <h1>I’m Mariane Glad to see you!</h1>
+         <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+         <h4>Let's connect</h4>
+         <div class="socials">
+            <img src="images/socials/Vector.svg"/>
+            <img src="images/socials/linkedin.svg"/>
+            <img src="images/socials/Vector (2).svg"/>
+            <img src="images/socials/Vector (3).svg"/>
+            <img src="images/socials/Vector (4).svg"/>
+         </div>
```
- [ ] For the social media icons, kindly made the following changes:

- The `div` tag with the class `socials` should be a `ul` tag. 
- Each image should be wrapped in a `li` tag and then `a` tag. Wrapping icons in `a` tag is not only good for semantic reasons but it maps users to your social media page through the `href` attribute of the `a` tag.

So the section should be like this:
```HTML
<ul>
  <li>
    <a href="twitter.com/username"><img src="images/socials/Vector.svg"/></a>
  </li>
...
</ul>
```

AyoMoses1 replied:
Thanks so much for this. I will implement asap

Gambit142 commented on styles.css:
```diff
@@ -0,0 +1,71 @@
+  margin: 2px 0;
+}
+
+#headline {
+  padding: 114px 24px;
+  background: url("images/Header\ bg@2x.png") no-repeat;
+  background-size: cover;
+  background-position: center top;
+}
```
- [ ] The `#headline` section should be aligned with flexbox. This gives us a lot of options to align all our elements in the stand-alone section plus, it's responsive.

Gambit142 commented on styles.css:
```diff
@@ -0,0 +1,71 @@
+
+#headline {
+  padding: 114px 24px;
+  background: url("images/Header\ bg@2x.png") no-repeat;
+  background-size: cover;
+  background-position: center top;
```
- [ ] The background image should be aligned properly. Aligning them properly according to the original design on Figma makes your app prettier. Kindly note the numbers 1, 2 and 3 in the image attached below and compare them to the background image you have. 

![screenshot](https://user-images.githubusercontent.com/77106962/204925518-1ddca0c6-be6e-4b00-81f5-948d281cfcdc.png)

#### Hint: Numbers 1 and 2 are missing in your app and number 3 is not properly aligned. 👍

Gambit142 reviewed (changes requested):
Hi @AyoMoses1 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- The pull request summary is descriptive ✔️ 
- Great effort making the app look presentable and pretty 👍 

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Going forward, kindly make your commit messages capitalized. For example, "Add index.html file" instead of "add index.html file" 👍  


_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider the optional changes I suggested in the comments under the review. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

AyoMoses1 committed:
Create License.md

This is a license file that is created for this project. As such, it describes and provides every information that describes its permission rights

AyoMoses1 committed:
Merge pull request #5 from AyoMoses1/add-license

Create License.md

Commit:
Update project from review

Commit:
Merge branch 'feature' of https://github.com/AyoMoses1/Portfolio-Redefined into feature

Commit:
update readme file

Commit:
fix stylelint errors

Commit:
Update readme file with social media handles

Commit:
Update .gitignore file

Commit:
Add project features in readme file

Commit:
Update style.css file

leonardodiasb reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @AyoMoses1 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - All linters checks passed ✔️
 - The pull request summary is descriptive ✔️
 - Great effort making the app look presentable and pretty 👍

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AyoMoses1 commented:
Thanks so much for the feedback
